dynamic-layers/openembedded-layer/recipes-devtools/bpftrace/bpftrace/0001-cmake-Bump-max-LLVM-version-to-19.patch

```diff
@@ -1,27 +1,24 @@
 From 16186113346c268a0bb45424ba1c41768b7e94cf Mon Sep 17 00:00:00 2001
 From: Khem Raj <raj.khem@gmail.com>
 Date: Fri, 16 Feb 2024 10:14:41 -0800
-Subject: [PATCH 4/4] cmake: Bump max LLVM version to 18+
+Subject: [PATCH] cmake: Bump max LLVM version to 19
 
-Upstream-Status: Submitted [https://github.com/bpftrace/bpftrace/pull/3009]
+Upstream-Status: Pending
```

Comment:
Was it un-submitted or just changed Upstream-Status by accident?

Reply:
the original patch was for clang18 which was merged. This is for clang19 which is not yet submitted.

Reply:
I have submitted it upstream and updated the patch